### ISSUE TITLE
Support for read_from_replicas option in deadpool_redis

### ISSUE THREAD

**Issue by fruscianteee** [bikeshedder/deadpool#350]
Hello,

I'm currently using the redis-rs crate, which provides a read_from_replicas option. When this option is enabled, read queries will be sent to the replica nodes, while write queries will be sent to the primary nodes.

I would like to know if there is a way to enable the read_from_replicas option when using deadpool_redis. If this feature is not currently supported, are there any plans to add support for it in the future? 

Thank you for your assistance.

**Comment by fruscianteee** [comment]
I believe it can be enabled with something like this:
```rust
let client = ClusterClientBuilder::new()
    .read_from_replicas()
    .build()?;
```
So, simply calling it on the builder should be sufficient.

> I'm always happy to accept PRs if you want to implement this yourself.

Thank you for the opportunity. This would be my first contribution, but I'd like to give it a try. I'll review the source code in `deadpool/redis/src/cluster` and come up with a proposal for the changes.

**Comment by fruscianteee** [comment]
Thank you for reviewing the PR. It was a great experience, and I really enjoyed it. I’ll be happy to contribute again if there’s anything else in the future.

With that, I’d like to close this issue.

**Comment by bikeshedder** [comment]
I just published `deadpool-redis 0.17.0` on crates.io including your changes:

- :package: https://crates.io/crates/deadpool-redis/0.17.0

Thanks a lot!
